src/scss/_themes.scss

```diff
@@ -19,39 +19,80 @@
 	}
 }
 
-@mixin oTeaserCollectionSpecial() {
+@mixin oTeaserCollectionSpecial($colour) {
 	position: relative;
 	border-top: 0;
 	margin-bottom: 50px;
 
 	&:after {
 		content: '';
 		position: absolute;
-		top: 0px;
+		top: 0;
 		left: 50%;
 		z-index: -1;
 		width: 100vw;
 		height: 100%;
-		background: $_o-teaser-collection-color-special-bg;
+		background: $colour;
 		transform: translateX(-50%);
 	}
 }
 
+@mixin oTeaserCollectionAssassination {
+	@include oTeaserCollectionBigStory();
+
+	.o-teaser-collection--assassination-related {
+		@include oTeaserCollectionSpecial($_o-teaser-collection-color-assassination-bottom-bg);
+		z-index: 1;
+		margin-bottom: 0;
+		padding-left: 10px;
+
+		&:after {
+			left: 52%;
+		}
+
+		// modifies borders when stacked(S) and horizontal(M+)
+		.o-teaser-collection__item {
+			margin: 0;
+			border-color: rgba(255, 255, 255, 0.2);
+			border-style: solid;
+			border-width: 0 0 1px;
+			padding: 15px 0;
+
+			@include oGridRespondTo(M) {
+				margin: 20px 0;
+				border-width: 0 1px 0 0;
```

Comment:
and likewise here just setting the `border-right` explicitly and `border-bottom: 0`.

Alternatively, instead of replacing everything, you could just go with `border-bottom-width: 1px` above and `border-bottom-width: 0; border-right-width: 1px` here. Shorthand properties can be quite hard to grok sometimes.